languages/en/filters.genie

```diff
@@ -180,3 +181,218 @@ range_with_filter = {
         ]);
     };
 }
+
+// npp filters
```

Comment:
I am ok with using abbreviations in the non-terminal name, but the comment is not very informative... Can you write down in the comment what each grammar category stands for, and maybe put the examples in?
I know everything is in the wiki, but people might not find the wiki at first...